### PR TITLE
Switch to forge

### PR DESCRIPTION
**Pull request by mostarz93**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -4,3 +4,12 @@
 [submodule "lib/forge-std"]
 	path = lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+[submodule "lib/openzeppelin-contracts"]
+	path = lib/openzeppelin-contracts
+	url = https://github.com/openzeppelin/openzeppelin-contracts
+[submodule "lib/chainlink"]
```
Comment (kibagateaux) [inline .gitmodules:10]: The chainlink submodule isn't setup properly but OZ installs fine. I think you said that there wasn't a forge package for chainlink already so you would try it manually, is this the manual submodule code?

**Comment by kibagateaux** [comment]
Can you keep the `package.json` file. It's good meta documentation on the project itself. I'll go through it and figure out which are the exact packages we can delete

File: .gitmodules (hunk 1 above)
Comment (mostarz93) [reply .gitmodules:10]: yeah it is.

**Comment by mostarz93** [comment]
> Can you keep the `package.json` file. It's good meta documentation on the project itself. I'll go through it and figure out which are the exact packages we can delete

done!

**Comment by kibagateaux** [comment]
We can delete `lib/ds-test/`, thats dapptools too.|
The chainlink git submodule is definitely not configured properly. The code doesnt download like OZ does and i get errors running tests that chainlink contracts dont exist

**Review by kibagateaux — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
[the remark this answers is not in the document]
Comment (mostarz93) [reply .gitmodules:10]: ok i resolved this. Gotta Needs to be

https://github.com/OpenZeppelin/openzeppelin-contracts.git

[the remark this answers is not in the document]
Comment (mostarz93) [reply .gitmodules]: and 
    path = chainlink
    url = https://github.com/smartcontractkit/chainlink.git

**Comment by mostarz93** [comment]
OK. I had to clean out some of the hidden git files and folders, but i was able to install all the dependencies via forge install including chainlink and open zeppelin.

**Pull request merged**